src/jetson/models/SSD/layers/functions/detection.py

```diff
@@ -1,7 +1,7 @@
 import torch
 from torch.autograd import Function
-from ..box_utils import decode, nms
-from data import voc as cfg
+from ....utils.box_utils import decode, nms
+from ...data import voc as cfg
```

Comment:
This is fine but absolute imports are recommended by the [pep8](https://www.python.org/dev/peps/pep-0008/#imports) standard citing "Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages)"